sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -201,10 +196,10 @@ public UserDto selectOrFailByLogin(DbSession session, String login) {
    * Please note that email is case insensitive, result for searching 'mail@email.com' or 'Mail@Email.com' will be the same
    */
   public boolean doesEmailExist(DbSession dbSession, String email) {
-    return mapper(dbSession).countByEmail(email.toLowerCase()) > 0;
+    return getMapper(dbSession).countByEmail(email.toLowerCase()) > 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define the locale to be used in this String operation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1449)